Source/CLI/CLI_Help.cpp

```diff
@@ -66,6 +66,28 @@ return_value Help(ostream& Out, const char* Name, bool Full)
         "    --merge-log value\n"
         "        Store merge log to value (file name).\n"
         "\n"
+        "    --merge-output-speed value\n"
```

Comment:
What is `value`? From code, this seems to be a flag rather than needing a value.